src/definition.ts

```diff
@@ -0,0 +1,12 @@
+import * as ReadonlyRecord from "fp-ts/ReadonlyRecord";
+import * as ReadonlyArray from "fp-ts/ReadonlyArray";
+import { flow } from "fp-ts/function";
+
+export const mapDefinitionsByName = (name: string) => flow(
+	ReadonlyRecord.toEntries,
+	ReadonlyArray.map(([key, value]) => [key.slice(0, name.length), value] as const),
+	ReadonlyRecord.fromEntries,
+);
```

Comment:
so, i think i actually found a bug or incompatibility with the current UI api.

my tooltips were not working as i'd expect from the code. after some debugging i realized it's because the hash we are passing into the API (a short hash) is not the hash that's returned (a long hash).

i solved this bug by just formatting the API response such that it uses short hashes for keys instead, based on the input name length.

i'm not familiar enough with unison to know if there's a better solution here. i'm open to suggestion.